PackagePrice.java

```diff
@@ -0,0 +1,52 @@
+package pricecomparison;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class PackagePrice {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		WebDriver driver = new FirefoxDriver();
+
+		String appUrl = "http://www.tripfactory.com/package/get-price-req?pkgId=27433&ahdate=06%2F05%2F2016";
+		driver.get(appUrl);
+
+		WebElement wE = driver.findElement(By.xpath(".//*[@id='invCalMth2']/table"));
+
+		System.out.println(wE.getText());
+
+		driver.findElement(By.xpath(".//*[@id='invCalMth2']/table/tbody/tr[2]/td[1]/div[2]/div[3]")).click();
+		System.out.println("selecting date: 1/05/2016");
+		driver.findElement(By.xpath(".//*[@id='calculateButton']/a")).click();

```

Comment:
And how can we use findElements here?FindElements returns a list of elements and here we are clicking the calculate button.